### ISSUE TITLE
Question about Line breaks on iOS

### ISSUE THREAD

**Issue by dmytro-fido** [JetBrains/compose-multiplatform#4335]
**Describe the bug**
Same CMP screen is used on Android and iOS (injected into SwiftUi)
On Android text breaks before word starts.
On iOS text breaks in the middle of word.
Example of usage:
```
    Text(
        text = text
    )
```
Neither setting softWrap for Text() composable or any of lineBreak: LineBreak options to TextStyle help.

**Affected platforms**
Select one of the platforms below:
- iOS

**Versions**
- Kotlin version*: 1.9.21
- Compose Multiplatform version*: 1.5.11
- OS version(s)* (required for Desktop and iOS issues): iOS 16.5.1
- OS architecture (x86 or arm64): arm64

**To Reproduce**
Just place quite long text inside Text() composable and display it on iOS

**Expected behavior**
Line breaks are made before word starts.

**Screenshots**
left iOS, right - Android
<img width="915" alt="image" src="https://github.com/JetBrains/compose-multiplatform/assets/126558920/7cfbde98-54d9-4cb3-bc54-6fe113f13eb8">

**Additional context**
On iOS it also doesn't respect custom breaks in text (with "\n"). It draws text in single line.

**Comment by dima-avdeev-jb** [comment]
Thanks for this issue! I will check it.

**Comment by dima-avdeev-jb** [comment]
@dmytro-fido
> On iOS it also doesn't respect custom breaks in text (with "\n"). It draws text in single line.
 I tested this code:
```Kotlin
Text("Some\ntext\nwith\nline\nbreaks")
```
And it works as expected.

**Comment by dmytro-fido** [comment]
@dima-avdeev-jb yeah, it's something weird. I tried on empty project and it works (both "\n" and line breaks for long text)
still can't see a difference between empty project and our app. Same plugin version. Tried launching our app with default material theme and default text styles. And it still doesn't work there.
I will continue investigations and will post here if I find root cause.

**Comment by dmytro-fido** [comment]
@dima-avdeev-jb hello again
So I found what caused line breaks work wrong
it's one of the pods we use:
```
pod 'MediaPipeTasksVision', '0.10.5'
```
I attached archive with minimum project setup to reproduce the issue 
[ViewDisappears.zip](https://github.com/JetBrains/compose-multiplatform/files/14386806/ViewDisappears.zip)

Besides that, when this pod is included, app crashes on clicking on TextField inside BottomSheet
Stack trace:
[ios_crash_log.txt](https://github.com/JetBrains/compose-multiplatform/files/14386860/ios_crash_log.txt)


In our real app, it doesn't crash, instead BottomSheet view disappears (It's hard to say what exactly happens. Keyboard is visible but the whole bottom sheet isn't). But, it was crashing on compose plugin 1.5.11 with same stack trace after user types text and then clicks backspace.
Should I open new ticket for this crash?

**Comment by dima-avdeev-jb** [comment]
Thanks! For now, I think this Issue is enought.

**Comment by dmytro-fido** [comment]
@dima-avdeev-jb just to let you know
I tried building iosApp with this hack
```
configurations.all {
    resolutionStrategy.eachDependency {
        if (requested.group.startsWith("org.jetbrains.skiko")) {
            useVersion("0.7.96")
        }
    }
}
```
from issue you've mentioned
and it worked. Line breaks work
when could we expect RC with skiko updated?
we of course can use dependency resolution workaround, but we afraid to forget to remove it once proper RC is released.

**Comment by dmytro-fido** [comment]
Fixed in 1.6.1

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-4335">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
